src/main/java/de/oliver/fancyholograms/commands/hologram/InsertAfterCMD.java

```diff
@@ -44,7 +44,17 @@ public boolean run(@NotNull CommandSender player, @Nullable Hologram hologram, @
         for (int i = 4; i < args.length; i++) {
             text += args[i] + " ";
         }
+
+        // peaqe
```

Comment:
You can remove those commands, your name will be visible in the git log and git blame for each line changed